src/os/bluestore/BlueStore.cc

```diff
@@ -7080,6 +7080,19 @@ int BlueStore::cold_close()
   return 0;
 }
 
+// derr wrapper to limit enormous output and avoid log flooding.
+// Of limited use where such output is expected for now
+#define fsck_derr(err_cnt, threshold) \
```

Comment:
i can imagine extending this in the future to take an argument for the type/class of message, so that we limit each type of message independently.  e.g., an counter for per_pool_omap.  we can do that when its time to limit the next message type tho